android/src/com/mapswithme/maps/widget/placepage/BannerController.java

```diff
@@ -390,9 +387,9 @@ private void fillViews(@NonNull MwmNativeAd data)
 
   private void animateActionButton()
   {
-    View view = mOpened ? mActionLarge : mActionSmall;
+    View view = isDetailsState(mState) ? mActionLarge : mActionSmall;
```

Comment:
This statement is a bit redundant as you do the same check one if-statement later. I would suggest to just use the if-else below to reduce the complexity of the code. This would mean that animator in the if becomes:
```
 animator = ObjectAnimator.ofObject(mActionLarge, "backgroundColor", new ArgbEvaluator(),
                                         colorFrom, colorTo, colorFrom);
```
and animator in the else becomes
```
animator = ObjectAnimator.ofFloat(mActionSmall, "alpha", 0.3f, 1f);
```

Reply:
Done.